qiskit/pulse/transforms.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Basic rescheduling functions which take schedules or instructions
```

Comment:
There is already a file that contains such transformations: `reschedule.py`. Please move this to that file

Reply:
Please see my recent PR #4174 @lcapelluto. I prefer the name transforms to `reschedule`. I will, of course, break this out into a separate PR but I believe this to be more canonical within a compiler.

Reply:
@lcapelluto, I understand if you temporarily want it to live in `reschedule`. However, I think this name is confusing.

Reply:
I'm all for it, but let's do this in another PR